priv/repo/migrations/20211209122351_create_invoices.exs

```diff
@@ -87,5 +91,16 @@ defmodule Siwapp.Repo.Migrations.CreateInvoices do
       add :items_id, references(:items)
       add :taxes_id, references(:taxes)
     end
+
+    create table(:templates) do
+      add :name, :string, size: 255
+      add :template, :text
+      add :print_default, :boolean, default: false
+      add :email_default, :boolean, default: false
+      add :subject, :string, size: 200
+      add :deleted_at, :utc_datetime
+
+      timestamps()
+    end
```

Comment:
this is not part of your code. Please do a rebase

Reply:
Sinceramente, no sé que es lo que pasa con esta rama. 
Simplemente hago:

- git pull (desde el main)
- git checkout "mi_rama"
- git rebase main

Y luego resuelvo conflictos. No sé porqué se me añaden los archivos que han modificado otros.

Reply:
pues si no puedes con rebase haz una rama nueva y empieza de cero, pero no subo mas que tu codigo